src/includes/constants.php

```diff
@@ -139,11 +139,11 @@
         ) {
             $requestURI = $_SERVER['REQUEST_URI'];
             // Check if /base/public or /base are in the request
-            foreach ([$baseURL, dirname($baseURL)] as $candidate) {
-                if (stripos($requestURI, $candidate) === 0) {
-                    return $candidate;
-                }
+        foreach ([$baseURL, dirname($baseURL)] as $candidate) {
```

Comment:
Appreciate you did this to fix php-cs, but yuck. One thing I’d check (if you haven’t already) is whether the failure is caused by the first `if` condition above being on the same line. I.e. change
```php
if ($baseURL
    && PUBLIC_DIR
```

to 

```php
if (
    $baseURL
    && PUBLIC_DIR
```

Reply:
Yeah, thanks @kinglozzer , that makes sense. I was having a hard time figuring out why it wanted that. I tried several reformats, but I'm not sure that was one of them.

It's only in the newer release of php-cs. Bizarre.

Reply:
Yeah, thanks, @kinglozzer . That makes sense. I could not figure out why it wanted that change. I tried several reformats, but I'm not sure that was one of them. Only in the newest release of php-cs, too. Bizarre.